AddBinary/main.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace DefaultNamespace
+{
+    public class main
+    {
+        public string AddBinary(string a, string b)
+        {
+            Stack<char> result = new Stack<char>();
+            
+            int lengthDiff = a.Length > b.Length ? a.Length - b.Length : b.Length - a.Length;
+            string appendZeros = new string('0',lengthDiff);
+            
+            if (a.Length > b.Length) b = appendZeros + b; 
```

Comment:
sometimes we may need the original value of the passed param. So in this case, I would define a new variable instead of changing the value of b directly. This will also give us an idea about the new value we assigned.

Reply:
After checking the code, can't we get rid of appending zeros? since it only adds extra iterations that are not needed.

Reply:
I'll see what i can do , 
it can be substituted with a loop at the end and extra condition in the first loop to check when a string is done